### PR TITLE
sync github actions workflows with latest IUC version

### PR DESCRIPTION
**Pull request by bernt-matthias**
Currently workflows do not run anymore, since github disabled `set-env`.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -3,17 +3,31 @@ on:
...
+    types: [run-all-tool-tests-command]
 env:
-  GALAXY_RELEASE: release_20.05
+  GALAXY_RELEASE: release_20.09
+  MAX_CHUNKS: 40
 jobs:
   setup:
     name: Setup cache
-    if: github.repository_owner == 'galaxyproteomics'
+    if: github.repository_owner == 'galaxyproject'
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply .github/workflows/ci.yaml:14]: Ist this repo owned by galaxyproject?

[the remark this answers is not in the document]
Comment (bgruening) [reply .github/workflows/ci.yaml:14]: nope

[the remark this answers is not in the document]
Comment (bernt-matthias) [reply .github/workflows/ci.yaml]: done. mixed up + and - :)

File: .github/workflows/slash.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Slash Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  slashCommandDispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Slash Command Dispatch
+        if: github.repository_owner == 'galaxyproject'
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply .github/workflows/slash.yaml]: done

**Pull request merged**